### PR TITLE
Whole Wheat Dough

### PR DESCRIPTION
**Pull request by Cheridan**
A vegan alternative to standard dough.

https://tgstation13.org/phpBB/viewtopic.php?f=10&t=3089

### CODE REVIEW

**Comment by TheVekter** [comment]
are you fucking serious

**Comment by RemieRichards** [comment]
@TheVekter you betcha

**Comment by babinsquared** [comment]
:+1: merge asap we can't afford to wait on this game-breaking oversight

**Comment by Atskadan** [comment]
nice

**Comment by MrPerson** [comment]
/r/ gluten free dough

File: code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm [hunk 1]
```diff
@@ -136,13 +136,13 @@ datum/reagent/consumable/milk/on_mob_life(var/mob/living/M as mob)
...
 	name = "Soy Milk"
 	id = "soymilk"
 	description = "An opaque white liquid made from soybeans."
 	color = "#DFDFC7" // rgb: 223, 223, 199
 
-datum/reagent/consumable/soymilk/on_mob_life(var/mob/living/M as mob)
+datum/reagent/consumable/milk/soy/on_mob_life(var/mob/living/M as mob)
 	if(M.getBruteLoss() && prob(20))
 		M.heal_organ_damage(1,0)
 	..()
```
Comment (phil235) [inline code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm]: this is going to stack the effects of milk and soymilk.

Comment (pudl) [inline code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Drinks.dm]: WOOP WOOP

**Comment by Jalleo** [comment]
Next we have gluten intolerarant disability

**Comment by phil235** [comment]
I have a competing PR for this, #8236.

**Comment by Cheridan** [comment]
omg wtf phil grudgecoding against me

ragequit

**Pull request closed without merging**